libr/syscall/syscall.c

```diff
@@ -83,9 +83,16 @@ R_API bool r_syscall_setup(RSyscall *s, const char *arch, int bits, const char *
 		s->regs = fastcall_mips;
 	} else if (!strcmp (arch, "avr")) {
 		s->sysport = sysport_avr;
-	} else if (!strcmp (os, "darwin") || !strcmp (os, "osx") || !strcmp (os, "macos")) {
+	} else if (!strcmp (os, "macos")) {
```

Comment:
Why did you remove darwin and the others?

Reply:
I added the darwin while goofing around and it got merged. The osx I cant find anywhere else in the codebase and was added as part of the first syscall pull request, I believe incorrectly, then later patched to add macos but osx wasnt removed.

Reply:
Looks like we use macos and darwin both internally (but again, no osx) https://github.com/radare/radare2/blob/master/libr/bin/format/mach0/mach0.c#L2120-L2125

I found some random older mac bins and some older stuff (though still 64 bit) did come in as darwin so thats still an exercised code path so Ill actually put my darwin check back in.

The only other use of darwin is in r2 egg? whats thats? Perhaps that file needs an ```|| !strcmp (os, "macos"``` as well??

Reply:
PRed that here https://github.com/radare/radare2/pull/11090